Cython/Includes/cpython/unicode.pxd

```diff
@@ -23,6 +28,22 @@ cdef extern from *:
     # New in version 3.3.
     Py_ssize_t PyUnicode_GET_LENGTH(object o)
 
+    Py_UCS1 *PyUnicode_1BYTE_DATA(object o)
+    Py_UCS2 *PyUnicode_2BYTE_DATA(object o)
+    Py_UCS4 *PyUnicode_4BYTE_DATA(object o)
+
+    # Deprecated since version 3.10, will be removed in version 3.12.
+    int PyUnicode_WCHAR_KIND
```

Comment:
Let's make it clear that the comment only belongs to one line, not the whole block.
```suggestion
    int PyUnicode_WCHAR_KIND  # Deprecated since version 3.10, will be removed in version 3.12.
```